config/jobs/kubernetes-client/haskell/client-haskell-presubmits.yml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-client/haskell:
+  - name: client-haskell-unit-tests
+    always_run: true
+    decorate: true
+    spec:
+      containers:
+      - image: ubuntu:bionic
```

Comment:
Image here also include the store to pull the image from, ex `gcr.io/project/image:tags`

Reply:
Specified image as `docker.io/ubuntu:bionic`